packages/calypso-build/bin/cb.js

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+if ( ! process.argv.some( arg => arg.startsWith( '--config' ) ) ) {
```

Comment:
Seeing `cb --config='./webpack.config.js'` in several places in this PR makes me think this could be a bit smarter.

Do you think we should check for the presence of `path.join( process.cwd() , 'webpack.config.js' )` and give that precedence over the package-provided one?

Reply:
Good question -- that would arguably be akin to what #32468 does for `babel.config.js`.

I was on the fence since I didn't want `cb` to behave that differently from Webpack CLI, or make it act too magically, but this might be a worthwhile change.

Reply:
> since I didn't want cb to behave that differently from Webpack CLI

That's actually why I mentioned it, I think the CLI will pick up config files ([more than just `./webpack.config.js`?](https://github.com/webpack/webpack-cli/blob/42ecb48df3fc0cc4977e8a258c6b7796e343e24e/bin/utils/convert-argv.js#L72-L84)), so using the local one would seem to be the least surprising. I don't think we need to reproduce the whole config resolution, but improving the most common case is probably beneficial.

Reply:
8755b59